linkml_model_enrichment/get_felix_tsv.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
move to synbio repo

Reply:
haven't moved yet, but have removed synbio mentions from this repo